articles/azure/azs-gs.md

```diff
@@ -59,6 +59,11 @@ Azure Active Directory is Microsoft's cloud-based Directory and Identity Access
 
 ## How can I get started with UKCloud for Microsoft Azure?
 
+> [!Important]
+> You agree that you will be required to enter into a direct Microsoft Cloud Solution Provider Customer Agreement. Once you have accepted the agreement we are obliged to report your name, email address and date of acceptance to Microsoft.
```

Comment:
Does the below describe the process a bit more clearly? I want to make it clear that all the customer does is provide their information to their CSM -- that is considered them accepting the agreement, right? They don't need to physically sign something? If there's more to the process than that, let me know and I'll come up with some different wording...

By providing the information below, you agree to enter into a direct Microsoft Cloud Solution Provider Customer Agreement (for more information see [here](https://docs.microsoft.com/en-us/partner-center/agreements) (Europe/United Kingdom)). If you accept the terms of the agreement, when you provide the information below, we're obliged to report your name, email address and date of acceptance to Microsoft.